### PR TITLE
[BIOMAGE-2017] Removed v1 references

### PR DESCRIPTION
**Pull request by kafkasl**
# Description

# Details
#### URL to issue
https://biomage.atlassian.net/browse/BIOMAGE-2017

#### Link to staging deployment URL (or set N/A)
https://ui-ahriman-flaky-warthog.scp-staging.biomage.net/

#### Integration test branch
master

# Merge checklist
Your changes will be ready for merging after all of the steps below have been completed.

### Code updates
Have best practices and ongoing refactors being observed in this PR
- [x] Migrated any selector / reducer used to the new format.

### Manual/unit testing
- [x] Tested changes using InfraMock locally **or** no tests required for change, e.g. Kubernetes chart updates.
- [x] Validated that current unit tests for code work as expected and are sufficient for code coverage **or** no unit tests required for change, e.g. documentation update.
- [x] Unit tests written **or** no unit tests required for change, e.g. documentation update.

### Integration testing
**You must check the box below** to run integration tests on the latest commit on your PR branch.
Integration tests have to pass before the PR can be merged. Without checking the box, your PR
**will not pass** the required status checks for merging.

- [ ] Started end-to-end tests on the latest commit.

### Documentation updates
- [ ] Relevant Github READMEs updated **or** no GitHub README updates required.
- [ ] Relevant Wiki pages created/updated **or** no Wiki updates required.

### Optional
- [ ] Staging environment is unstaged before merging.
- [ ] Photo of a cute animal attached to this PR.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -17,9 +17,14 @@ endif
...
 #--------------------------------------------------
 install: 
+	@echo "Installing local runner"
 	@(cd ./local-runner && npm install)
+	@echo "Installing R env packages"
+	@(cd ./pipeline-runner && R -e "renv::restore()")
 build: 
 	@(cd ./local-runner && npm run build)
+test:
+	@(cd ./pipeline-runner && R -e "devtools::test()")
```
Comment (gerbeldo) [inline Makefile:27]: is this working? we had some problems with @cosa65 running the install script.

**Review by gerbeldo — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (kafkasl) [reply Makefile:27]: it works for me but I had most of the deps already installed. We can merge and debug it later

Comment (gerbeldo) [reply Makefile:27]: yep, I agree. we solved it by manually installing the specific packages that were failing

Comment (kafkasl) [reply Makefile:27]: can those missing packages be installed also with a command? like `R -e "install.packages(...)`

File: pipeline-runner/init.R [hunk 2]
```diff
@@ -65,6 +65,7 @@ load_config <- function(development_aws_server, api_version = "v1") {
         pod_name = Sys.getenv("K8S_POD_NAME", "local"),
         activity_arn = activity_arn,
         api_url = paste0("http://api-",sandbox,".api-",sandbox,".svc.cluster.local:3000"),
+        api_version = "v2",
```
[the remark this answers is not in the document]
Comment (kafkasl) [reply pipeline-runner/init.R:68]: it's only ever used in this same function to build the SNS topic name. I was considering removing it altogether

**Pull request merged**